seqr/views/apis/staff_api_tests.py

```diff
@@ -576,37 +524,69 @@ def test_success_story(self):
         self.assertEqual(len(response_json['rows']), 1)
         self.assertDictEqual(response_json['rows'][0], EXPECTED_SUCCESS_STORY)
 
-    @mock.patch('seqr.views.apis.staff_api.export_multiple_files')
+    @mock.patch('seqr.views.utils.export_utils.zipfile.ZipFile')
```

Comment:
This PR only changes the output of the sample metadata, and not the anvil download. However, the input to `export_multiple_files` for the anvil export changed, even though these change had no impact on the files in the end. Since what is really important with the anvil metadata is that the contents of the zipfile remained unchanged, I don't want the test to fail every time an intermediate step looks different, I only want it to fail if the final output changes. Therefore, I updated this test to mock out the lower level function and check the actual file content, rather than mocking out the higher level function